app_flutter/lib/service/fake_cocoon.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:fixnum/fixnum.dart';
+
+import 'package:cocoon_service/protos.dart'
+    show Commit, CommitStatus, Stage, Task;
+
+import 'cocoon.dart';
+
+class FakeCocoonService implements CocoonService {
+  @override
+  Future<List<CommitStatus>> getStats() {
+    return Future.delayed(Duration(seconds: 1), () => _getFakeStats());
```

Comment:
Don't induce artificial delays in the code. Try using the `Future.value` constructor:

```suggestion
    return Future.value( _getFakeStats());
```

Reply:
Done